src/components/SEO.js

```diff
@@ -10,17 +10,21 @@ const METADATA = [
   },
 ];
 
-const DocsSiteSeo = ({ location, title }) => (
+const DocsSiteSeo = ({ location, title, description }) => (
   <SEO location={location} title={title}>
     {METADATA.map((data) => (
       <meta key={data.name} {...data} />
     ))}
+
+    {description ||
+      (title && <meta name="description" content={description || title} />)}
```

Comment:
```suggestion
    {(description || title) && <meta name="description" content={description || title} />}
```

I think prettier got ya 😄. The parens should wrap `description || title` to ensure the boolean logic runs correctly, otherwise you'll only get this meta tag when `title` is present

Reply:
You may want to make that change locally as I'm not sure the above suggestion is prettier friendly

Reply:
Ugh yeah that's prettier.

Sorry, I'm moving much too quickly and making a lot of stupid mistakes.